src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -110,16 +110,31 @@ object DepositHandler {
     }
 
     val payload = Paths.get(settings.tempDir.toString, id, deposit.getFilename.split("/").last).toFile
-    for {
+    val depositDir = Paths.get(settings.tempDir.toString, id).toFile
+    val receipt: Try[DepositReceipt] = extractPayloadAndGetDepositReceipt(deposit, contentLength, payload, depositDir)
+    setFilePermissions(settings, id, depositDir) // ensure all files in deposit are accessible for the deposit group
+    receipt
+  }
+
+  private def setFilePermissions(settings: Settings, id: DepositId, depositDir: File) = {
```

Comment:
It would really help us in the future if you add return types for private methods as well. I know it is not necessary for the Scala compiler, but for us 'humans' it is quite nice to see what the return type is.

(same for the method below)